docs/sources/panels/visualizations/pie-chart-panel.md

```diff
@@ -6,9 +6,9 @@ weight = 850
 
 # Pie chart
 
-The pie chart displays reduced series, or values in a series, from one or more queries, as they relate to each other, in the form of slices of a pie. The arc length, area and central angle of a slice are all proportional to the slices value, as it relates to the sum of all values. This type of chart is best used when you want a quick comparison of a small set of values in an aesthetically pleasing form.
+{{< figure src="/static/img/docs/pie-chart-panel/pie-chart-example.png" max-width="1200px" lightbox="true" caption="Table visualization" >}}
```

Comment:
Should `caption="Pie chart vizualization"`? Also, if small things like this are ok for me to fix directly I'm happy to do so.

Edit, made the fix.